test/bdd/bddtests_test.go

```diff
@@ -83,7 +85,7 @@ func TestMain(m *testing.M) {
 					panic(fmt.Sprintf("Error composing system in BDD context: %s", err))
 				}
 
-				testSleep := 120
+				testSleep := 180
```

Comment:
why increase to 180?

Reply:
I was doing this locally for some testing - I didn't mean to commit this. I reverted back to 120.